core/src/main/java/org/neo4j/ogm/cypher/compiler/builders/node/DefaultRelationshipBuilder.java

```diff
@@ -77,6 +77,7 @@ public String type() {
     }
 
     @Override
+    @SuppressWarnings("HiddenField")
```

Comment:
Why no rename as in the other changes above?
(this comment is applicable for quite all other changes that contain `@SuppressWarnings("HiddenField")` besides test classes)